testdata/BUILD

```diff
@@ -244,6 +244,25 @@ container_image(
 #     ],
 # )
 
+container_image(
+    name = "with_unix_epoch_creation_time",
+    base = ":base_with_volume",
+    creation_time = "1234567890000",
+)
+
+container_image(
+    name = "with_rfc_3339_creation_time",
+    base = ":base_with_volume",
+    creation_time = "1989-05-03T12:58:00Z",
+)
+
+container_image(
+    name = "with_stamped_creation_time",
+    base = ":base_with_volume",
+    creation_time = "{BUILD_TIMESTAMP}",
```

Comment:
I sort of wonder whether we want to allow the flexibility of choosing a particular workspace status variable, or if we should just always use `BUILD_TIMESTAMP` when stamping is enabled.

Reply:
That's kind of what I had in mind as well.

Reply:
`SOURCE_DATE_EPOCH` is the standard proposed by Reproducible Builds: https://reproducible-builds.org/specs/source-date-epoch/

It makes sense to me to standardize on `BUILD_TIMESTAMP` since that's likely what other rules might use as well.

Reply:
I feel that giving the option to specify a different workspace status variable is unnecessary.

I don't have a strong opinion about allowing hard-coded values here, though at least for Kubernetes we plan to just set `SOURCE_DATE_EPOCH` for reproducibility (and cacheability).

Reply:
I'm fine with allowing users to set this, but let's default to `{BUILD_TIMESTAMP}` when `stamp=True` so that they don't have to?

Reply:
Works for me. Done.